src/kfactory/kcell.py

```diff
@@ -6616,6 +6630,8 @@ def clean_value(
         return "_".join(clean_value(v) for v in value)
     elif isinstance(value, dict):
         return dict2name(**value)
+    elif isinstance(value, BaseModel):
+        return str(hash(value))
```

Comment:
BaseModel has no proper implementation of `__hash__` afaik. So if you were to do:

```python
@kf.cell
def test_cell(model: MyModel) -> kf.KCell:
    c = kf.KCell
    return c

c1 = test_cell(my_model)
c2 = test_cell(my_model.model_copy())

kf.kcl.write(...)
```
will error out because hash(my_model) != hash(my_model.model_copy())